lib/lotus/routing/endpoint_resolver.rb

```diff
@@ -197,7 +198,7 @@ def classify(string)
       private
       def resolve_callable(callable)
         if callable.respond_to?(:call)
-          @endpoint_class.new(callable)
+          create_endpoint(callable, @endpoint_class)
```

Comment:
@AlfonsoUceda We can review it later, when the PR for Controller will be in place ;)